src/Data/Enum/Gen.purs

```diff
@@ -0,0 +1,16 @@
+module Data.Enum.Gen where
+
+import Prelude
+
+import Control.Monad.Gen (class MonadGen, chooseInt)
+import Data.Enum (class BoundedEnum, fromEnum, toEnum)
+import Partial.Unsafe (unsafePartial)
+import Data.Maybe (fromJust)
+
+-- | Create a random generator for a finite enumeration. `toEnum i` must be
+-- | well-behaved: it must return a `Just` value for all `Int`s between
+-- | `fromEnum bottom` and `fromEnum top`.
+genBoundedEnum :: forall m a. MonadGen m => BoundedEnum a => m a
+genBoundedEnum =
+  unsafePartial fromJust <<< toEnum
```

Comment:
By `bottom .. top`, I meant to say `[enumFromTo](https://pursuit.purescript.org/packages/purescript-enums/3.0.0/docs/Data.Enum#v:enumFromTo) bottom top`, which must be safe, I think.